src/ed_infrastructure/utils/jwt/jwt_handler.py

```
@@ -0,0 +1,46 @@
+            return jwt.encode(
+                dict(payload),
+                self._secret,
+                algorithm=self._algorithm,
+            )
+        except Exception as e:
+            LOG.error(f"Error encoding jwt payload: {payload}. {e}")
+            raise ApplicationException(
+                Exceptions.InternalServerException,
+                "Internal server error.",
+                ["Internal server error."],
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)